web/styles/modal.css

```diff
@@ -219,7 +219,7 @@
 
 #read_receipts_modal {
     .modal__container {
-        width: 360px;
+        width: 25.7142em; /* 360px at 14px em */
```

Comment:
Should all the pixel-valued paddings, `right` value, and similar things in this section be converted? My guess is they'll be fairly subtle but would make it a more consistent design across sizes.

Reply:
I've generally been focusing on the pieces that look particularly broken (which sometimes includes padding but not always). My opinion is that if the screenshots look fine then it's fine to not convert the padding, and better to focus on other higher-impact info density changes.